src/pilot/templates/dell-environment.yaml

```diff
@@ -33,7 +33,23 @@ parameter_defaults:
   # NovaVcpuPinSet: ['8-47']
  
   # Number of Dell Compute nodes 
-  DellComputeCount: 3
+
+  DellComputeCount: 0
```

Comment:
Is there a compelling reason for changing DellComputeCount from 3 to 0?  I would prefer it to remain 3 since that is what our standard install uses.